client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -545,9 +539,10 @@ class Checkout extends React.Component {
 	}
 
 	handleTermChange = ( { value: planSlug } ) => {
-		this.getPlanProducts().forEach( removeItem );
```

Comment:
I think removing this line introduced a slight regression - there's a flash of two products now when you switch between 1 and 2 year versions:
![2018-04-25 14_06_03](https://user-images.githubusercontent.com/3392497/39244808-1ee5f1a4-4892-11e8-9e58-f12852c74c0f.gif)

The reason being, without explicitly removing the plan, you rely on the backend to remove the extra plan - which requires a request to the backend and as a result a temporary state where two plans are in the cart.

Reply:
Thanks, I restored it!